.gitlab-ci.ornl.yml

```diff
@@ -0,0 +1,80 @@
+
+stages:
+  - build
+  - test
+  - clean
+
+Spack Build:
+  stage: build
+  tags:
+    - nobatch
+  needs: []
+  variables:
+#    CUSTOM_CI_BUILDS_DIR: "/gpfs/wolf/proj-shared/csc449/ci/${CI_PIPELINE_ID}"
+    WORK: "/gpfs/wolf/proj-shared/csc449/ci/"
+  script:
+    - pwd
+    - env | grep CI_
+    - export run_id=$CI_PIPELINE_ID
```

Comment:
Love this, will make this change to the llnl pipeline!

Reply:
Yeah, not sure how that works for multiple machines - but for now it's quite convenient to have the collected.  Ok to leave it as such until we get a second ORNL machine included?